monkeytype/encoding.py

```diff
@@ -195,13 +195,15 @@ def __init__(
         qualname: str,
         arg_types: str,
         return_type: Optional[str],
-        yield_type: Optional[str]
+        yield_type: Optional[str],
+        caller: Optional[str]
```

Comment:
Can we make this default to `None` for better backward-compatibility with existing third-party extensions of MonkeyType that may be instantiating `CallTraceRow`?